src/runner.ts

```diff
@@ -278,12 +250,12 @@ export class StaticAnalysisParserRunner {
     }
 
     private getPath(result: sarifReportTypes.ReportResult): string {
-        return result.locations[0].physicalLocation.artifactLocation.uri;
+        return result.locations[0]?.physicalLocation?.artifactLocation?.uri;
     }
 
     private getLine(result: sarifReportTypes.ReportResult): number {
-        const {region} = result.locations[0].physicalLocation;
-        return region.startLine ?? region.endLine;
+        const {region} = result.locations[0]?.physicalLocation;
```

Comment:
There is a eslint issue:
<img width="1275" height="322" alt="image" src="https://github.com/user-attachments/assets/c854325d-25dc-404f-9316-1d79ac77771d" />
Change it to: `const region = result.locations[0]?.physicalLocation?.region;`